### PR TITLE
Making an Invoice page

### LINKED ISSUE #238: 🫰Invoice page
We need a page to administer invoices. Let's create a page and move on from there.

### PR DESCRIPTION
**Pull request by jazellemaira**
This PR creates an InvoicePage where clients can see a list of their invoices. Fixes #238  

Screenshot 2024-10-18 at 14 04 48

I created a new issue #293  for making the layout of individual invoices.

### CODE REVIEW

**Comment by nsunami** [comment]
@jazellemaira Shall we merge this one, although the individual invoice page is not yet ready?

**Comment by jazellemaira** [comment]
@nsunami  Yes, I think so. individual invoice needs to be a separate PR.

File: web/src/pages/InvoicePage/InvoicePage.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import { Download, FileSearch, FileText } from 'lucide-react'
+import { Button } from 'web/src/components/ui/button'
+
+import { Metadata } from '@redwoodjs/web'
+
+const InvoicePage = () => {
+  return (
+    <>
+      <Metadata title="Invoice" description="Invoice page" />
+      <div className="relative -left-4 -top-4 min-h-[120vh] w-screen bg-white">
```
Comment (nsunami) [inline web/src/pages/InvoicePage/InvoicePage.tsx:11]: Are these class names intentional?

Comment (jazellemaira) [reply web/src/pages/InvoicePage/InvoicePage.tsx]: Made some changes to address this question. ☺️

**Comment by jazellemaira** [comment]
Here's the latest version:

Screenshot 2024-11-20 at 15 13 09

**Review by nsunami — APPROVED** [review]
LGTM!

**Pull request merged**